go/iot/main.go

```diff
@@ -93,22 +200,68 @@ func initOpenCensus(projectId string, reportPeriod int) {
 	}
 	view.RegisterExporter(exporter)
 
-	// Set reporting period to report data at every second.
+	// Set reporting period to report data based on the given reportPeriod.
 	view.SetReportingPeriod(time.Second * time.Duration(reportPeriod))
 
-	// Create view to see the processed video size cumulatively.
+	// Create view to see the sound strength distribution.
+	// Subscribe will allow view data to be exported.
+	// Once no longer need, you can unsubscribe from the view.
+	if err := view.Register(&view.View{
```

Comment:
nit: consider defining these views as constants, similar to soundStrengthDistMeasure, etc.

Reply:
Sounds great